LDAR_Sim/simulations/simple_test_case1/M_OGI.yaml

```diff
@@ -4,8 +4,8 @@ method_name: OGI
 deployment_type: mobile
 measurement_scale: component
 sensor:
-  type: OGI_camera_zim
-  minimum_detection_limit: [0.24, 0.39] 
+  type: OGI_camera_rk
```

Comment:
Did you mean to change this?

Reply:
Nope, it was changed while making sure the sensor worked and got pushed through. will change back.